<?xml version="1.0"?>
<launch>
    <!--  run kinect-->
  <include file="$(find freenect_launch)/launch/freenect.launch">
	<arg name="depth_registration" value="true" />
  </include>

  <node pkg="apriltags_ros" type="apriltag_detector_node" name="apriltag_detector">
  <rosparam param="tag_descriptions">[{id: 1, size: 0.07}]</rosparam>
  <remap from="/image_rect" to="/camera/rgb/image_rect_color"/>
  <remap from="/camera_info" to="/camera/rgb/camera_info"/>
  </node>

    <!--  add the obj,  tag, grasp frame
  <node name="KUKA_add_frame" pkg="agent_KUKA_robot" type="KUKA_add_frame.py" respawn="false" output="screen" />
-->
<!--
  <node name="camera_positioner" pkg="camera_positioner" type="camera_positioner" respawn="false" output="screen" />
-->
</launch>
